nextflow.config

```diff
@@ -98,6 +98,7 @@ params {
 
     // centrifuge
     run_centrifuge             = false
+    centrifuge_kreport         = false
```

Comment:
```suggestion
```

Not necessary, I think this should run by default if centrifuge is executed (it should be a small enough process I think)